clang/test/CodeGenHLSL/builtins/select.hlsl

```diff
@@ -0,0 +1,41 @@
+// RUN: %clang_cc1 -finclude-default-header -x hlsl -triple \
+// RUN:   dxil-pc-shadermodel6.3-library %s -emit-llvm -disable-llvm-passes \
+// RUN:   -o - | FileCheck %s --check-prefixes=CHECK
+
+// CHECK: %hlsl.select = select i1
+// CHECK: ret i32 %hlsl.select
```

Comment:
`select i1` doesn't seem sufficient, we should probably check all of the argument types. Also, we can't rely on the SSA value names - they won't necessarily be there in release builds. We can solve both of these things with some regex:
```c++
// CHECK: [[SELECT:%.*]] = select i1 {{%.*}}, i32 {{%.*}}, i32 {{%.*}}
// CHECK: ret i32 [[SELECT]]
```